### ISSUE TITLE
Can't override SendbirdSdk options

### ISSUE THREAD

**Issue by marianoo-andres** [sendbird/Sendbird-Flutter#11]
Hi,

We need to increase the file upload timeout limit because the default of 30 seconds is too low for our requiriments. I've tried with not luck overriding SendbirdSdk options to 300 seconds but it seems to have no effect.

Currently the only workaround we found is to edit the default options directly on Sendbird SDK codebase. Obviously this is a hack and we would prefer not to touch the library.

### Where it was tested
Mac OS X Catalina
iOS Simulator with iOS 13.7
Flutter 2.0.4
Dart 2.12.2

Flutter doctor output:
```
[✓] Flutter (Channel stable, 2.0.4, on Mac OS X 10.15.6 19G2021 darwin-x64, locale en-AR)
    • Flutter version 2.0.4 at /Users/marianoandres/Flutter/flutter
    • Framework revision b1395592de (4 weeks ago), 2021-04-01 14:25:01 -0700
    • Engine revision 2dce47073a
    • Dart version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/marianoandres/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[!] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.7, Build version 11E801a
    ! Xcode 11.7.0 out of date (12.0.1 is recommended).
      Download the latest version or update via the Mac App Store.
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 49.0.2
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] Connected device (2 available)
    • iPhone 8 (mobile) • 4B38CE84-3B3A-42F1-A4B2-DB40E4D19247 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-13-7 (simulator)
    • Chrome (web)      • chrome                               • web-javascript • Google Chrome 90.0.4430.85

! Doctor found issues in 1 category.
```

### Steps to reproduce:
1) Add options parameter when initializing SendbirdSdk
```
final options = Options(fileTransferTimeout: 300, websocketTimeout: 300, connectionTimeout: 300);
final sendbird = SendbirdSdk(appId: "asddddsasaddsdsdsadsadsa", options: options);
```

2) Upload a big file and it should timeout after 30 seconds (the default value)

With my internet connection the video below takes around 45 seconds to be uploaded, so it times out. You may need to use a larger  file.

https://we.tl/t-dBN3Rvd1TY

Thank you.

**Comment by intoxicated** [comment]
Will look into this and let you know when it gets fixed. Thanks for the report

**Comment by intoxicated** [comment]
FYI I am planning to release next patch around late this week or early next week.

**Comment by intoxicated** [comment]
@marianoo-andres fix has been released with 3.0.12
